### ISSUE TITLE
Code Action at file end

### ISSUE THREAD

**Issue by jrieken** [microsoft/vscode#83932]
* open https://github.com/microsoft/vscode/blob/ec7e8633bff79de6cd12266d0c510921096e9b99/src/vs/editor/contrib/referenceSearch/referenceSearch.ts#L214
* reveal last line
* :bug: a code action on the trail newline suggests to convert functions to async

<img width="369" alt="Screenshot 2019-11-04 at 16 52 11" src="https://user-images.githubusercontent.com/1794099/68135185-8f063e00-ff23-11e9-8ba6-c4f49ddad5c1.png">

**Comment by mjbvz** [comment]
Not seeing this with TS 3.7.1-rc:

<img width="277" alt="Screen Shot 2019-11-04 at 9 01 26 AM" src="https://user-images.githubusercontent.com/12821956/68141077-b9cfa280-fee1-11e9-9ba5-e9fb91fbe150.png">

What TS version is enabled in your workspace?

**Comment by jrieken** [comment]
YEah, using 3.7.1-rc. Did you try this with commit ec7e8633bff79de6cd12266d0c510921096e9b99?

**Comment by mjbvz** [comment]
Still not seeing this. Does this go away if you edit the file? Are there any errors in your dev tools? I saw an error related to markers earlier today but have not been able to reproduce the issue

**Comment by jrieken** [comment]
Hard to say.. I have moved on this refactoring this file, having said that I did make a lot of changes in that file and it might haven gotten stale for reason?
